### PR TITLE
enable forge deployment

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,3 +47,14 @@ branches:
     - /^v\d/
 notifications:
   email: false
+deploy:
+  provider: puppetforge
+  username: puppet
+  password:
+    secure: "whG1hlsH4KREOkHopzdPRc965y005VzSwTpSzFw9ho30QSvvwv/jAJ7izbkrYwaTHGlOC5S1Wk3k0UsuVp0NE69zErnOFplqHo0BLI40uuhTCQQ4Qm6i/+44AtmjtFGO//cWs4fr6yyVC9kyIvsP+3rx/qt6GNxoltbNkvb7Hco="
```
Comment (frizop) [inline .travis.yml:54]: I mistakenly closed the review, but yeah I don't think we want to tell anybody who pulls down this module what your deploy key is

**Review by frizop — CHANGES_REQUESTED** [review]
It might be a good idea to remove the secure value and then do something like the following:

    git update-index --skip-worktree .travis.yml

That way you can put your secure value back in, and it won't track those changes.

File: .travis.yml (hunk 1 above)
Comment (jhoblitt) [reply .travis.yml:54]: eh? This is a travis secure value that's tied to this repo.

Comment (frizop) [reply .travis.yml:54]: oh.. that's somewhat confusing to me. So is the secure value for travis or puppet forge? Also, can you confirm that there is no requirement around making sure this value is secure, stored in git, etc?

Comment (jhoblitt) [reply .travis.yml:54]: It is a pgp encrypted secret of my forge passphrase.  Only travis has the private key and its tied to this repo. See https://docs.travis-ci.com/user/encryption-keys/

Comment (frizop) [reply .travis.yml:54]: oh perfect! Was just making sure nothing got mixed up there.

**Review by frizop — APPROVED** [review]

**Pull request merged**